slicedimage/_typeformatting.py

```diff
@@ -17,7 +17,7 @@ def format_tile_dimensions(tile_dimensions):
     return result
 
 
-def format_imagepartition_dimensions(imagepartition_dimensions):
+def format_tileset_dimensions(imagepartition_dimensions):
```

Comment:
arg should be named tileset_dimensions? or partition_dimensions?

Reply:
Will fix.